app/controllers/katello/api/v2/errata_controller.rb

```diff
@@ -36,7 +36,7 @@ def available_for_content_view_filter(filter, collection)
 
     def custom_index_relation(collection)
       collection = filter_by_cve(params[:cve], collection) if params[:cve]
-      hosts = ::Host::Managed.authorized("view_hosts")
+      hosts = ::Host::Managed.authorized("view_hosts").where(:organization_id => params[:organization_id])
```

Comment:
organization_id is not a required parameter, so we probably need an if check and only apply the where() if its set.